packages/machine-extractor/src/__tests__/actions.test.ts

```diff
@@ -0,0 +1,1178 @@
+import { extractMachinesFromFile } from '../extractMachinesFromFile';
+
+function getTestMachineConfig(configStr: string) {
+  const result = extractMachinesFromFile(configStr);
+  const machine = result!.machines[0];
+  const config = machine?.toConfig({ serializeInlineActions: true })!;
+
+  return config;
+}
+
+describe('extract actions', () => {
+  it('Ensures actions are extracted with the correct `kind`', () => {
```

Comment:
Overall I'd prefer to have more granular tests. I feel like it comes important when debugging things - it's easier to debug a single small test than to debug a noisy one when one of the tested cases in it doesn't work alright.

Not a blocker rn, just want to encourage a different style of writing test cases than this "catch-all" style